README.md

```diff
@@ -47,6 +48,16 @@ The `ntripping` utility has the following usage:
 Different resources can be requested from different locations. By default, a San
 Francisco latitude, longitude, and height will be used.
 
+The `--url` must be formatted as such: `{username}:{passsword}@{url}`, 
+where `username` and `password` refer to the user's NTRIP username and password.
```

Comment:
username and password are optional - some casters don't require accounts. And you can specify a username without a password and a password without a username.

`[[{username}]:[{passsword}]@]{url}`